ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/manager/ChainState.java

```diff
@@ -88,7 +88,7 @@ public void updateForAnnouncedBlock(
     }
   }
 
-  private void updateHeightEstimate(final long blockNumber) {
+  public void updateHeightEstimate(final long blockNumber) {
     estimatedHeightKnown = true;
     if (blockNumber > estimatedHeight) {
       estimatedHeight = blockNumber;
```

Comment:
This will need a `synchronized` block around the whole method content.  Previously it was only called from other methods which already had `synchronized`.

Reply:
done.